release/train_tests/trainium/compute_aws.yaml

```diff
@@ -0,0 +1,73 @@
+cloud_id: cld_ywvPyX4hezrjv8RtrU28wTbr
```

Comment:
what is this cloud? should we add some doc here on how this cloud is provisioned? or where does it come from?

like if some of the subnets here are broken or something, what should one do?

Reply:
This is the `test-efa-vpn-cloud`, we need EFA setup to run Neuron code.

Reply:
can you add the related info in the form of yaml comments?

like how to create / maintain this cloud and its related networks, and where to find it.

Reply:
Ah we are going to delete this release test, as we've decided to make AWSNeuron Integration as community supported.

Reply:
Can we resolve this then @woshiyyya @aslonnie ?

Reply:
Yes. The test is removed.